diff --git a/thirdparty/ffmpeg-decoder-rs/Cargo.toml b/thirdparty/ffmpeg-decoder-rs/Cargo.toml
index e46b4b2..004040e 100644
--- a/thirdparty/ffmpeg-decoder-rs/Cargo.toml
+++ b/thirdparty/ffmpeg-decoder-rs/Cargo.toml
@@ -20,7 +20,7 @@ default = []
 rodio_source = ['rodio']
 
 [dependencies]
-ffmpeg-sys-next = { version = "4.4.0", default-features=false, features=["avcodec", "avformat", "swresample", "static"] }
+ffmpeg-sys-next = { version = "4.4.0", default-features=false, features=["avcodec", "avformat", "swresample"] }
 
 thiserror = "1.0"
 log = "0.4"
